### PR TITLE
made cert chart

### PR DESCRIPTION
**Pull request by smark88**

### CODE REVIEW

File: charts/gcp-managed-cert/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+version: 0.1.0
+
+appVersion: 0.1
```
Comment (danxmoran) [inline charts/gcp-managed-cert/Chart.yaml]: Can we leave this out? It's not required, and it doesn't really mean anything in this context.

Comment (danxmoran) [reply charts/gcp-managed-cert/Chart.yaml]: Or else maybe set it to "networking.gke.io/v1beta1" to match the API version

File: charts/gcp-managed-cert/values.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+domainName: ""
```
Comment (danxmoran) [inline charts/gcp-managed-cert/values.yaml]: I would much rather have a `values.schema.json` for every chart in `monster-helm`, and only have `values.yaml` for meaningful defaults.

File: charts/gcp-managed-cert/templates/cert.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: networking.gke.io/v1beta1
+kind: ManagedCertificate
+metadata:
+  name: {{ .Release.Name }}
+  domains:
```
Comment (danxmoran) [inline charts/gcp-managed-cert/templates/cert.yaml:7]: Missing `spec:` between `name` and `domains`, I think

**Review by danxmoran — CHANGES_REQUESTED** [review]

File: charts/gcp-managed-cert/templates/cert.yaml (hunk 3 above)
Comment (smark88) [reply charts/gcp-managed-cert/templates/cert.yaml:7]: it's cut pasted from the argo-server chart

File: charts/gcp-managed-cert/values.yaml (hunk 2 above)
Comment (smark88) [reply charts/gcp-managed-cert/values.yaml]: Defining it all in the values.yaml seems to be the community standard way of showing a user what can be overwritten (even if blank) in all the stable charts. This way someone can use `helm show values monster/<chart>` to see what is defined or able to be overwritten.

Comment (danxmoran) [reply charts/gcp-managed-cert/values.yaml]: I am ok with not replicating that standard. For the charts here, I'd much rather have the up-front validation that blocks even rendering a chart when values are missing.

File: charts/gcp-managed-cert/templates/cert.yaml (hunk 3 above)
Comment (danxmoran) [reply charts/gcp-managed-cert/templates/cert.yaml:7]: That one has a `spec:`: https://github.com/broadinstitute/monster-helm/blob/master/charts/argo-server/templates/service.yaml#L19-L25

File: charts/gcp-managed-cert/values.yaml (hunk 2 above)
Comment (smark88) [reply charts/gcp-managed-cert/values.yaml]: So are you asking me to delete values.yaml and just write a schema?

Comment (danxmoran) [reply charts/gcp-managed-cert/values.yaml]: Yes please 😄 Blame the Scala boi in me for wanting the static checks

**Review by danxmoran — APPROVED** [review]
:shipit:

**Review by raaidbroad — APPROVED** [review]

**Review by quazi-h — APPROVED** [review]

**Pull request merged**